sympy/combinatorics/schur_number.py

```diff
@@ -0,0 +1,120 @@
+"""
+The Schur number S(k) is the largest integer n for which the interval [1,n]
+can be partitioned into k sum-free sets.(http://mathworld.wolfram.com/SchurNumber.html)
+"""
+import math
+from sympy import Symbol
+from sympy.core import S
+from sympy.core.function import Function
+
+
+class schur_number_lower_bound(Function):
+    """
+    Gives the value for 1/2 * (3**n  - 1)
```

Comment:
I don't see the point of this either. This is such a simple expression you could just use it directly wherever it is needed (it looks like it isn't actually used elsewhere in this module).

Reply:
It is for the user to get the lower bound of ```S(n)``` i.e the schur number.